### PR TITLE
V5.0.0 release candidate

### PR DESCRIPTION
**Pull request by capsulecorplab**

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+* update derivedFrom in ViewBeaconData user story
+* append detail to behavior in user story 2
+* remove user story 2.1
+* update derivedFrom in health data structure
+* update beacon to refer to health or mission data
+* replace payload user stories for scheduling with updating mission mode duration
+* update user story 3 to include any interrupt timer
+* bump major version no. in package.json
+* add user story for requesting to set MM duration
+* remove user stories 6-6 for payload scheduler
```
Comment (kchiu002) [inline CHANGELOG.adoc:25]: Didn't you remove all the user stories starting with 5?

**Review by kchiu002 — COMMENTED** [review]
Capitalize the start of each bullet point to match the rest of the log.
See line item comment as well.

File: CHANGELOG.adoc (hunk 1 above)
Comment (capsulecorplab) [reply CHANGELOG.adoc]: Correct, which was then replaced w/ 5-SetMissionModeDuration.yaml

**Review by kchiu002 — APPROVED** [review]
LGTM

**Pull request merged**